brainspell/test_tornado.py

```diff
@@ -131,28 +134,51 @@ def test_procfile():
         assert filename in os.listdir()
 
 
-""" TODO: get selenium testing working
-def test_existence(): #Using selenium testing
+
+def test_existence(): #Using selenium testing to verify existence of site elements
     driver.get("https://brainspell.herokuapp.com")
-    assert "Brainspell" in driver.title #Checks website title is accurate
+    driver.implicitly_wait(3)
+    assert "Brainspell" in driver.title #Checks website was correctly received
 
     #Ensuring elements of view article page are all present
     driver.get("https://brainspell.herokuapp.com/view-article?id=00000000")
+    WebDriverWait(driver,3).until(
+        expected_conditions.presence_of_element_located((By.CLASS_NAME,"dropbtn"))
+    )
     meshButtons = driver.find_elements_by_class_name("dropbtn")
+
     assert meshButtons != None #Ensure Mesh terms are included
```

Comment:
assert meshButtons
(no need to say != None; also, None is a singleton, so you should never be using != with it.)